docs/src/prototypes/alerts/BannerAlerts.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import {
+  Flex,
+  Provider,
+  RadioGroup,
+  ThemeAnimation,
+  RadioGroupItemProps,
+  Header,
+  Button,
+} from '@stardust-ui/react'
+
+import AnimatedBannerAlert from './AnimatedBannerAlert'
+import ComposeMessage from '../chatPane/composeMessage'
+
+type BannerName = 'info' | 'oof' | 'danger' | 'urgent' | 'critical'
+
+const isAlertClosable = (bannerName: BannerName) =>
+  bannerName !== 'critical' && bannerName !== 'urgent'
+
+const getBannerContent = (bannerName: BannerName) => `${_.startCase(bannerName)} banner`
+
+const bannerNames: BannerName[] = ['info', 'oof', 'danger', 'urgent', 'critical']
+
+const bannerRadioItems: RadioGroupItemProps[] = bannerNames.map(bannerName => ({
+  key: bannerName,
+  value: bannerName,
+  label: getBannerContent(bannerName),
+}))
+
+const slideDown: ThemeAnimation = {
+  keyframe: {
+    from: { transform: 'translateY(0)' },
+    to: { transform: 'translateY(100%)', display: 'none' },
+  },
+  duration: '.3s',
+  iterationCount: '1',
+  timingFunction: 'linear',
+  fillMode: 'forwards',
+}
+
+interface BannerAlertsState {
+  selectedBannerName: string
+  open: boolean
+}
+
+class BannerAlerts extends React.Component<{}, BannerAlertsState> {
+  state = {
+    selectedBannerName: bannerRadioItems[0].value as BannerName,
+    open: true,
+  }
+
+  openSelectedBanner = () => this.setState({ open: true })
+
+  closeSelectedBanner = () => this.setState({ open: false })
+
+  render() {
+    const { selectedBannerName, open } = this.state
+
+    return (
+      <Provider theme={{ animations: { slideDown } }}>
+        <Flex
+          column
+          style={{
+            width: '50%',
+            backgroundColor: '#f3f2f1',
+            padding: '10px 50px',
+            position: 'relative',
+          }}
+        >
+          <Flex space="between" vAlign="center">
+            <Header as="h3">Select a banner:</Header>
+            <Button content="Reset Banner" onClick={this.openSelectedBanner} />
+          </Flex>
+          <RadioGroup
+            checkedValue={selectedBannerName}
+            items={bannerRadioItems}
+            checkedValueChanged={(e, { value }) =>
+              this.setState({ selectedBannerName: value as BannerName, open: true })
+            }
+          />
+          <AnimatedBannerAlert
+            {...{
+              key: selectedBannerName,
+              attached: true,
+              [selectedBannerName]: true,
+              content: getBannerContent(selectedBannerName as BannerName),
+              ...(isAlertClosable(selectedBannerName as BannerName) && {
+                open,
+                action: { icon: 'close', onClick: this.closeSelectedBanner },
+              }),
+            }}
+          />
+          <ComposeMessage attached="bottom" />
```

Comment:
It looks strange that we need to add some additional prop to the component that we want the Alert to be attached to.

Reply:
I actually think it makes sense because the `ComposeMessage` has rounded borders at the top by default so we're attaching the components to each other.

It's done the same way in: https://react.semantic-ui.com/collections/message/#variations-attached

![image](https://user-images.githubusercontent.com/5442794/54769334-46527c80-4c01-11e9-8620-5f67bd01cebf.png)

thoughts?

Reply:
found solution with @mnajdova and @kuzhelov thanks guys, resolving